### ISSUE TITLE
Hard to find method from list

### ISSUE THREAD

**Issue by nathanshammah** [unitaryfund/metriq-app#419]
As a user, when I want to add a method to a submission, I find it hard to parse the existing methods' list from the modal window. See this screenshot for reference:
<img width="872" alt="methods-metriq" src="https://user-images.githubusercontent.com/14573436/153403638-6070ede0-74dd-43d3-8cb4-9e63878338e7.png">
I propose two possible solutions:
- autocomplete, as for tags
- alphabetical list

or a mix of the two.

**Comment by WrathfulSpatula** [comment]
For the moment, I think an alphabetically sorted drop-down is better. Soon, this might change, but it works for now.
